bliss/encoder.py

```diff
@@ -163,20 +180,51 @@ def _encode_ptiles(self, image_ptiles: Tensor, n_samples: Optional[int]):
                 galaxy_bools = (
                     torch.rand_like(galaxy_probs) > 0.5
                 ).float() * is_on_array.unsqueeze(-1)
+
             tile_samples.update(
                 {
                     "galaxy_bools": galaxy_bools,
                     "galaxy_probs": galaxy_probs,
                 }
             )
 
+            if self.lensing_binary_encoder is not None:
+                assert not self.lensing_binary_encoder.training
+                lensed_galaxy_probs = self.lensing_binary_encoder.forward(image_ptiles, locs)
+                lensed_galaxy_probs *= is_on_array.unsqueeze(-1)
+
+                if deterministic:
+                    lensed_galaxy_bools = (
+                        lensed_galaxy_probs > 0.5
+                    ).float() * is_on_array.unsqueeze(-1)
+                else:
+                    lensed_galaxy_bools = (
+                        torch.rand_like(lensed_galaxy_probs) > 0.5
+                    ).float() * is_on_array.unsqueeze(-1)
```

Comment:
Black auto formatting isn't so great for long lines. It's often clearer to create a temporary/intermediate variable to reduce the length of long lines